bundles/org.openhab.binding.ojelectronics/src/main/java/org/openhab/binding/ojelectronics/internal/config/OJElectronicsBridgeConfiguration.java

```diff
@@ -40,7 +40,7 @@ public class OJElectronicsBridgeConfiguration {
     /**
      * Url for API
      */
-    public String apiUrl = "https://OWD5-OJ001-App.ojelectronics.com/api";
+    private String apiUrl = "https://OWD5-OJ001-App.ojelectronics.com";
```

Comment:
See comment below.
```suggestion
    private final String apiUrl = "https://OWD5-OJ001-App.ojelectronics.com";
```

Reply:
I think this is not possible because this is an binding configuration parameter. If a make it final the binding configuration is ignored at apiUrl has always it's default value